test/libsolidity/semanticTests/array/push/array_push_nested_from_memory.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
+    uint8 b = 23;
+    uint120[][] s;
+    uint8 a = 17;
+    function f() public returns(uint120) {
+        uint120[] memory m = new uint120[](3);
+        m[0] = 1;
+        s.push(m);
+        assert(s.length == 1);
```

Comment:
This only works for the first call to this function though

Reply:
Added delete at the beginning of a function now